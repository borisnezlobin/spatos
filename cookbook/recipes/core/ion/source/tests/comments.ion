echo Hello world  # End of line comments are ignored

#echo Goodbye world
	#echo Nada
echo tabs ok	#comment
echo not#a#comment
