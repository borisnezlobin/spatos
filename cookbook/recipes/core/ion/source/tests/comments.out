Hello world
tabs ok
not#a#comment
